### ISSUE TITLE
[📍] "Extra test arguments" is easy to confuse with "Extra Chrome arguments"

### ISSUE THREAD

**Issue by dave-2** [catapult-project/catapult#4274]
Someone ran a try job with extra Chrome arguments in the "Extra test arguments" field. But these are Telemetry arguments, not Chrome arguments. I think right now you can do Chrome args by putting `--extra-browser-args="foo"` Maybe we want a separate field for "Extra Chrome arguments"?

Do we even need a field for Telemetry arguments? The [Telemetry docs](https://github.com/catapult-project/catapult/blob/master/telemetry/docs/run_benchmarks_locally.md#benchmark-commands) don't describe any that would be useful in Pinpoint.

https://pinpoint-dot-pinpoint-dot-chromeperf.appspot.com/job/1444b343840000

@anniesullie @simonhatch

**Comment by anniesullie** [comment]
I just ran `tools/perf/run_benchmark help run` and saw quite a few options users might be interested in:

 * `--pageset-repeat`: I've definitely seen people do this in perf try jobs
 * `--story-tag-filter`: Some teams are actively using tagging, I can definitely imagine them wanting to run on specific types of content they've tagged.
 * `--browser-logging-verbosity` and `--no-performance-mode` seem important for troubleshooting
 * `--extra-chrome-categories` and `--extra-atrace-categories` seem really important for debugging. Right now those are automatically set through the "Request a debug trace" button, but they need to be easy to set everywhere.
 * `--use-live-sites`: is it okay if users do this?
 * `--results-label`: We should probably document how/whether this works with pinpoint

So I think:
 * We should definitely have a section for Chrome flags (ideally links to the list, I think [this one?](https://peter.sh/experiments/chromium-command-line-switches/))
 * We should definitely have a section for telemetry flags (ideally has a tooltip or other help to tell you to run `tools/perf/run_benchmark help run`)
* We should consider UX for trace categories in the future.

**Comment by quisquous** [comment]
There is also some weirdness with quotes in the extra test arguments field.

I put in this text, exactly: `--extra-browser-args="--enable-gpu-rasterization --enable-oop-rasterization --use-gl=angle"` for https://pinpoint-dot-chromeperf.appspot.com/job/14acf3f3840000.

And the link on that job says `extra_test_args ["--extra-browser-args=--enable-gpu-rasterization --enable-oop-rasterization --use-gl=angle"]`

**Comment by simonhatch** [comment]
Yeah we use --extra-chrome-categories and --extra-atrace-categories from the dashboard to change what's enabled for tracing. Also have had someone enable profilers and then retrieve the results (eventually they'll be surfaced via diagnostics).

**Comment by dave-2** [comment]
https://pinpoint-dot-chromeperf.appspot.com/job/13f4ee62440000

**Comment by dave-2** [comment]
https://pinpoint-dot-chromeperf.appspot.com/job/15cbb811440000
